index.html

```diff
@@ -1,6 +1,57 @@
-<head>
-  <script type="text/javascript" src="https://firefoxux.github.io/SharedBanner/fxux-banner.min.js"></script>
```

Comment:
Yes, it makes sense to have it, agree! Would be ok to have a non-js-generated banner as well?